modules/channel-web/src/views/lite/components/messages/renderer/FileMessage.tsx

```diff
@@ -18,8 +19,8 @@ export const FileMessage = (props: Renderer.FileMessage) => {
 
     extension = validUrl.pathname
   } catch (error) {
-    // If the URL is not valid return a dummy component.
-    return null
+    // Try using path.extname since url might be relative.
+    extension = path.extname(url)
```

Comment:
Probably, the carrousel element needs to have the same logic ? https://github.com/botpress/botpress/blob/master/modules/channel-web/src/views/lite/components/messages/renderer/Carousel.tsx#L68

The carousel only display image.